### DISCUSSION TITLE
Unable to group by repo or assignee on new project beta list view

### DISCUSSION THREAD

**Discussion by sawan-webmavens** [community/community#5964] [Projects and Issues] [answered]
It would be great to be able to view all issues grouped by assignee or by repo.  Right now we are able to group by milestone only.  but the milestones could be different for different repos.

**Comment by alexcnichols** [comment, marked as answer]
:wave: Marking this as a duplicate in favor of:

- https://github.com/github/feedback/discussions/6136
- https://github.com/github/feedback/discussions/5217
